0.preprocess-sites/4.image-and-segmentation-qc.py

```diff
@@ -164,31 +166,30 @@
     .reset_index()
     .rename(columns={0: "Ratio"})
 )
-
+empty_df=empty_df[['Percent_Empty']].stack().to_frame().reset_index().rename(columns={0: "Percent Empty"})
```

Comment:
I am not sure how else you are using `empty_df`, but if possible, you should try to keep data processing of a single variable all in one spot. What I mean is move this line closer to where you define `empty_df` (line 155).

This will help reader understanding a lot.